projects/tiempoCarlaSergio/src/views/Sobremi.vue

```diff
@@ -1,6 +1,16 @@
 <template>
-  <div id="sobremi" class="mt-2">
-    <p>Página a nivel nacional que muestra el tiempo</p>
+  <div id="sobremi" class="mt-2 text-justify">
+    <p>La información meteorológica elaborada a partir de la actividad científica
+      y técnica desenvolvida por la Conserjería de Medio Ambiente, Territorio y Vivienda,
+      Tiempo, acorde con el prograso científico y tecnológico es objeto de propiedades intelectuales
+      y goza de la protección reconocida por Texto Refundido de la Lei de Propiedad Intelectual aprobado
+      por Real Decreto Legislativo 1/1996 de 12 de abril.Protección de datos</p>
+      <div class="text-center">
+        <p class="font-weight-bold">TIEMPO &copy;</p>
+        <p> <span class="font-italic">Teléfono:</span> (+34) 881 999 654</p>
+        <p><span class="font-italic">Email:</span> tiempo@tiempo.es</p>
```

Comment:
Aunque ahora al usar otra API no tiene mucho sentido este texto, como curiosidad, en datos de contacto podéis usar enlaces con atributos para lanzar el cliente de correo o a "realizar una llamada":

```
<p><span class="font-italic">Teléfono:</span> <a href="tel:34 881 999 65">(+34) 881 999 654</a></p>
<p><span class="font-italic">Email:</span> <a href="mailto:tiempo@tiempo.es">tiempo@tiempo.es</a></p>
```